README.md

```diff
@@ -148,23 +148,24 @@ The functions will be loaded in the same order as they are inside the array.
 -------------------------------------------------------
 <a name="after"></a>
 
-### app.after(func([context], [done]), [cb])
+### app.after(func(error, [context], [done]), [cb])
```

Comment:
I do not think `after()`  should be called with an error at all. We should bail off the plugin loading.